src/taco-dependency-installer/test/tacoErrors.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * ******************************************************
+ *                                                       *
+ *   Copyright (C) Microsoft. All rights reserved.       *
+ *                                                       *
+ *******************************************************
+ */
+
+/// <reference path="../../typings/mocha.d.ts" />
+
+"use strict";
+
+import mocha = require ("mocha");
+import path = require ("path");
+
+import resources = require ("../resources/resourceManager");
+import testUtil = require ("../../taco-utils/testUtils/testHelper");
```

Comment:
Are we sure we want to require cross-package files using relative paths like this? When this is installed via NPM install, taco-utils most likely won't be in the same root dir as taco-dependency-installer... It will be under ./node_modules/taco-utils, so I believe this will make these tests fail